aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -72,10 +72,12 @@ class CAFFE2_API Dispatcher final {
   struct OperatorDef final {
```

Comment:
btw, just noticed - maybe we should rename it from OperatorDef to avoid confusion with Caffe2? Maybe OperatorEntry?

Reply:
I think OperatorEntry is less descriptive, so will keep it as OperatorDef now. But this is a private class inside of the Dispatcher and will never be visible outside of the Dispatcher class.